index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a <dfn data-abbr>verifiable digital credential</dfn> (VDC).
```

Comment:
```suggestion
      This document defines an API enabling a website to request presentation of a <dfn data-abbr>digital credential</dfn> (VDC).
```

Is "verifiable" necessary here? Can't we assume that a property of a digital credential is that it is verifiable? I'm a little worried this will be read as equivalent to a W3C VC (which is not, e.g. in the case of an MDoc?).

Reply:
verifiable digital credentials is being used as the umbrella term in many corners of the industry to cover all flavors including W3C VCs, mdocs, SD-JWT VCs, etc. 

If there is concern, we can add a more formal definition.

Reply:
> > Verifiable digital credentials is being used as the umbrella term in many corners of the industry to cover all flavors including W3C VCs, mdocs, SD-JWT VCs, AnonCreds, etc.
> 
> Ah, interesting, I didn't quite get that impression (but I'm certainly not as well connected). I get the impression that everybody seems to be aligning more on "digital credentials" than "verifiable digital credentials". For example, is the OpenID4VP WG using "verifiable digital credentials"?

Hey Sam - we have started using VDC as the go to term. Mostly because it adequately describes the characteristics of what we are looking for. There are many existing digital credentials that are no cryptographically verifiable (think traditional login credentials, etc.) and the term VC is very tightly associated with a specific data model. VDC is distinct and reasonably descriptive. That doesn't mean it HAS to be used here, but we would encourage its use. See this for more on it:

https://www.nist.gov/blogs/cybersecurity-insights/digital-identities-getting-know-verifiable-digital-credential-ecosystem

Reply:
```suggestion
      This document defines an API enabling a website to request presentation of a  [=digital credentials=].
```

Reply:
filed https://github.com/WICG/digital-credentials/issues/198 in case we want to enhance out definition of a "digital credential" to explicitly talk about being **verifiable**.